packages/gatsby/static/configuration/yarnrc.json

```diff
@@ -432,6 +425,13 @@
       "type": "string",
       "default": "pnp"
     },
+    "winLinkType": {
+      "_package": "@yarnpkg/plugin-pnp",
+      "description": "Applies to Windows only. If set to `junctions` Yarn will use Windows junctions when linking workspaces into `node_modules` directories, which are always absolute paths. If set to `symlinks` Yarn will use symlinks, which will use relative paths,and is consistent with Yarn's behavior on non-Windows platforms.",
```

Comment:
This description doesn't give hints why Yarn doesn't use `symlinks` on Windows by default. On Windows only users having `create symbolic links` privilege can create symlinks, whereas `junctions` can be created by any user.

Reply:
Updated. If this isn't clear enough, please suggest or make edits. TBH, I had never heard of Junctions before working on this enhancement (I am much more familiar with *nix).